core/merge.js

```diff
@@ -178,7 +178,7 @@ class Merge {
     if (!file.sides.remote) {
```

Comment:
I forgot about adding a log call at the beginning of the function to help debug cases that use it.